proto/cosmos/crypto/ed25519/keys.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+package cosmos.crypto.ed25519;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/cosmos/cosmos-sdk/crypto/keys/ed25519";
+
+// PubKey defines a ed25519 public key
+// Key is the compressed form of the pubkey. The first byte depends is a 0x02 byte
+// if the y-coordinate is the lexicographically largest of the two associated with
+// the x-coordinate. Otherwise the first byte is a 0x03.
+// This prefix is followed with the x-coordinate.
+message PubKey {
+    option (gogoproto.goproto_stringer) = false;
```

Comment:
do we need this line?

Reply:
yes so we can implement our custom `String()` method https://github.com/cosmos/cosmos-sdk/blob/marie/6928-baseacc-pk/crypto/keys/ed25519/ed25519.go#L176